addons/auth_oauth/res_config.py

```diff
@@ -9,9 +9,18 @@
 class base_config_settings(osv.TransientModel):
     _inherit = 'base.config.settings'
 
+    def get_uri(self, cr, uid, context=None):
+        return "%s/auth_oauth/signin" % (self.pool['ir.config_parameter'].get_param(cr, uid, 'web.base.url', context=context))
+
+    def _get_server_uri(self, cr, uid, ids, name, arg, context=None):
+        return dict.fromkeys(ids, self.get_uri(cr, uid, context))
+
     _columns = {
         'auth_oauth_google_enabled' : fields.boolean('Allow users to sign in with Google'),
         'auth_oauth_google_client_id' : fields.char('Client ID'),
+        'server_uri_gmail': fields.function(_get_server_uri, type='char', string='Server uri'),
+        'module_auth_oauth': fields.boolean('Use external authentication providers (OAuth)'),
```

Comment:
@rga-odoo it's become very complex if we inherit view an add xpath, for the simplicity i redefine 'module_auth_oauth' field.